package.json

```diff
@@ -2,5 +2,8 @@
   "devDependencies": {
     "husky": "^5.0.6",
     "prettier-plugin-java": "^1.0.1"
+  },
+  "scripts": {
+    "postinstall": "husky install"
```

Comment:
Did you both have chance to look at `pretty-quick` as well.  The [Prettier docs](https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick) are pretty useful as a starting point.  You should be able to remove the `.husky` directory and just add the following json to your `package.json`:

```json
"husky": {
    "hooks": {
      "pre-commit": "pretty-quick --staged"
    }
  }
```

This will will inform husky of how to manage the pre-commit hook, essentially will format all staged files and means you won't have to do the install of husky yourself as it should recognise the `husky` keyword in the `package.json` and manage that for you 😄 

I would worth also be worth looking at adding in a script for formatting the whole codebase if we need to that at a later, so something like:

```json
"scripts": {
    "format": "prettier --write './**/*.{java,md,json,yml}'"
}
```

Which would you then be able to run `npm run format`